integration-cli/docker_utils.go

```diff
@@ -1353,3 +1353,33 @@ func createTmpFile(c *check.C, content string) string {
 
 	return filename
 }
+
+func buildImageArgs(args []string, name, dockerfile string, useCache bool) (string, error) {
+	id, _, err := buildImageWithOutArgs(args, name, dockerfile, useCache)
+	return id, err
+}
+
+func buildImageWithOutArgs(args []string, name, dockerfile string, useCache bool) (string, string, error) {
```

Comment:
WithOut referred to WithOutput following from `buildImageWithOut`, or with output.  One of the questions was whether the existing methods should be enhanced in a separate PR or these that include the `Args` should be included here and supplement to existing ones.